packages/mdc-icon-toggle/mdc-icon-toggle.scss

```diff
@@ -19,15 +19,17 @@
 @import "@material/animation/functions";
 @import "@material/ripple/mixins";
 @import "@material/theme/mixins";
+@import "./mixins";
 
 .mdc-icon-toggle {
-  @include mdc-theme-prop(color, text-secondary-on-light);
   @include mdc-ripple-surface;
   @include mdc-ripple-radius;
   @include mdc-states(black);
+  @include mdc-icon-toggle-ink-color(text-secondary-on-light);
```

Comment:
Is this intentionally using secondary-on-light, while the dark variant is using primary-on-dark?

Reply:
Not sure - I just blindly copy/pasted the original values.

Changed `text-secondary-on-light` to `text-primary-on-light`.